isic_archive/models/image.py

```diff
@@ -68,6 +78,29 @@ def createEmptyImage(self, originalFileRelpath, parentFolder, creator, dataset,
 
         return image
 
+    def resetImageForIngest(self, image):
+        originalFile = self.originalFile(image)
+        keepImageId = originalFile['_id'] if originalFile else None
+        attrsToDelete = [
```

Comment:
It looks like for images created along the normal pipeline, these fields are initialized in varying ways:
* `ingested` [is set to `False`](https://github.com/ImageMarkup/isic-archive/blob/a5fc734b62180b2fe2c2cb025671dc71a64fc0b6/isic_archive/models/image.py#L66) for every image upon creation
* `readable` is not set upon creation, then is set to `True` or `False` in `ingestImage`
* `ingestImage` is not set upon creation, then is set [a `dict` of `None` values](https://github.com/ImageMarkup/isic-archive/blob/a5fc734b62180b2fe2c2cb025671dc71a64fc0b6/isic_archive/tasks/image.py#L91) in `ingestImage`
* `largeImage` is not set upon creation, then is set to [a `dict` of fully populated values](https://github.com/ImageMarkup/isic-archive/blob/a5fc734b62180b2fe2c2cb025671dc71a64fc0b6/isic_archive/tasks/image.py#L214) in `generateLargeImage`
* `superpixelsId` is not set upon creation, then is set to an ObjectId in `generateSuperpixels`

By just deleting all these fields here, we are rolling back the image to a state that it typically doesn't have in a normal pipeline. I'm not sure if this will cause any bugs until the image fields are fully re-populated. A more thorough approach is to initialize all of these to `None` / empty values upon image creation and simply reset them to that same state here, so we have assurance that the fields always exist.